src/localisation/string_ids.h

```diff
@@ -2453,6 +2453,52 @@ enum {
 	STR_CHEAT_DISABLE_TRAIN_LENGTH_LIMIT_TIP = 5811,
 	
 	STR_SHORTCUT_SHOW_MULTIPLAYER = 5812,
+	
+	STR_HARDWARE_DISPLAY_TIP                  = 5813,
```

Comment:
Strings in string_ids.h are not tab-aligned. It's more consistent if you either
- change the tabs between the constant and `=` operator to spaces, like the rest.
- align the whole file with tabs (using a tab width of 4 spaces, not 8 as you did below).